code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -319,49 +332,39 @@ paths:
           $ref: '#/components/responses/403'
         '404':
           $ref: '#/components/responses/404'
-        '409':
-          description: Conflict
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/ErrorInfo'
-              example:
-                status: 409
-                code: CONFLICT
-                message: "App with a running application instance cannot be deleted"
         '500':
           $ref: '#/components/responses/500'
         '503':
           $ref: '#/components/responses/503'
  
-  /edge-cloud-nodes:
+  /edge-cloud-zones:
```

Comment:
There's some inconsistency between the paths `/app`, `/appinstance`, and `/edge-cloud-zones` in terms of singular/plural and hyphen/nohyphen. I believe the preferred convention for REST APIs is to use the plural version of nouns, and at least here to prefer no hyphenation, so we should have API paths of `/apps`, `/appinstances`, and `/edgecloudzones`. I have no preference, but it should at least be consistent.

Reply:
We think is important to maintain /edge-cloud-zones so we keep the harmonization with Simple Edge Discovery. Regarding hyphenation, we resolved in the last updated of the PR:
![image](https://github.com/camaraproject/EdgeCloud/assets/150795747/3c0b583a-c959-492f-a522-5a11341d17f1)

Looking for consistency, we should move to `/apps` and `/apps/{appId}/instances` but we are not sure if this could be confusing about the operation that the method is defined for.

Reply:
There are lots of articles if you search for "REST API singular vs plural". In general there is no right answer, although most recommend plural. But all agree to pick one convention and stick to it with all APIs. It looks like other APIs in CAMARA are using plural, so this spec should also use plural to be consistent.

Reply:
Agree on _pick one approach and stick to it_.

Reply:
Perfect, ` /apps` and `/apps/{appId}/instances`  will be the paths. And for future paths, plural is the approach to be followed.